### PR TITLE
v3 - customer case detail and landing page

### PR DESCRIPTION
**Pull request by mathiazom**
Initial components to replace the very basic landing page list and JSON detail view... 

### Detail page
image

### Landing page
image

### CODE REVIEW

File: src/components/customerCases/customerCase/CustomerCase.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import { CustomerCase as CustomerCaseDocument } from "studioShared/lib/interfaces/customerCases";
+
+import styles from "./customerCase.module.css";
+
+export interface CustomerCaseProps {
+  customerCase: CustomerCaseDocument;
+}
+
+export default function CustomerCase({ customerCase }: CustomerCaseProps) {
+  console.log(customerCase.image);
```
Comment (anemne) [inline src/components/customerCases/customerCase/CustomerCase.tsx:13]:
```suggestion
```

File: src/components/customerCases/customerCase/CustomerCase.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+          {customerCase.basicTitle}
+        </Text>
+        <div className={styles.mainImageWrapper}>
+          <SanitySharedImage image={customerCase.image} />
+        </div>
+        <div className={styles.ingress}>
+          <Text type={"bodyLarge"}>{customerCase.description}</Text>
+          <div className={styles.projectInfo}>
+            <div className={styles.projectInfoItem}>
+              <Text>Kunde</Text>
```
Comment (anemne) [inline src/components/customerCases/customerCase/CustomerCase.tsx:27]: Should not be hardcoded

File: studioShared/schemas/documents/customerCase.ts [hunk 3]
```diff
@@ -51,11 +51,6 @@ const customerCase = defineType({
       type: "array",
       of: [richTextBlock, imageBlock, listBlock],
     }),
-    defineField({
```
Comment (anemne) [inline studioShared/schemas/documents/customerCase.ts:54]: 👌

**Comment by mathiazom** [comment]
Merging this now, and then we can look at translating static content in a different PR

**Pull request merged**